### PR TITLE
Rename RecurringTasks.cs to Agent.cs

### PR DESCRIPTION
**Pull request by isaac73**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/Agent.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ private void buildDevicePropertiesCache() {
                 this.log.info("DeviceProperties cache created");
                 return;
             } catch (Exception e) {
-                this.log.warn("DeviceProperties creation failed, will retry in few seconds", e);
+                this.log.debug("DeviceProperties creation failed, will retry in few seconds");
```
Comment (dluc) [inline app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/Agent.java]: why this change?

Comment (isaac73) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/Agent.java]: This is too verbose when storage adapter is not available. There is already enough info in logs from here - > https://github.com/Azure/iothub-manager-java/blob/master/app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java

Comment (dluc) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/Agent.java]: how often does it occur? looks like a useful warning, which by the way should include the number of seconds, to be more informative

Comment (isaac73) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/Agent.java]: Every 10 seconds. We can keep it as "warn", but adding the exception "e" to log is unnecessary.
